.github/workflows/dotnet-build.yml

```diff
@@ -76,12 +76,18 @@ jobs:
     - name: list available kernels
       run: |
         python -m jupyter kernelspec list
-    - name: Setup .NET
+    - uses: astral-sh/setup-uv@v3
```

Comment:
Do we still need to setup uv if not running intergration tests in this job?

Reply:
yes because of the build requirements for the test